File: src/main/java/quizapi/model/SessionAnswer.java
package quizapi.model;

import javax.persistence.*;

@Entity
@Table(name = "session_answers")
public class SessionAnswer {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", unique = true, nullable = false)
    private Long id;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "game_session_id", nullable = false)
    private GameSession gameSession;

    @ManyToOne(targetEntity = Answer.class)
    private Answer answer;

    public SessionAnswer() {
    }

    public SessionAnswer(GameSession gameSession, Answer answer) {
        this.gameSession = gameSession;
        this.answer = answer;
    }

    public Long getId() {
        return id;
    }

    public GameSession getGameSession() {
        return gameSession;
    }

    public void setGameSession(GameSession gameSession) {
        this.gameSession = gameSession;
    }

    public Answer getAnswer() {
        return answer;
    }

    public void setAnswer(Answer answer) {
        this.answer = answer;
    }
}

File: src/main/kotlin/com/shkila/quizapi/model/SessionAnswer.kt
package com.shkila.quizapi.model

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.FetchType
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.JoinColumn
import javax.persistence.ManyToOne
import javax.persistence.Table

@Entity
@Table(name = "session_answers")
data class SessionAnswer(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        @Column(name = "id", unique = true, nullable = false)
        val id: Long? = null,

        @ManyToOne(fetch = FetchType.EAGER)
        @JoinColumn(name = "game_session_id", nullable = false)
        var gameSession: GameSession? = null,

        @ManyToOne(targetEntity = Answer::class)
        var answer: Answer? = null
)